repositories/like_repository.py

```
@@ -0,0 +1,61 @@
+    def __init__(self):
+        self.file_path = "data/likes.json"
+        # 데이터 폴더가 없으면 생성
+        os.makedirs(os.path.dirname(self.file_path), exist_ok=True)
+        # 파일이 없으면 빈 리스트로 초기화
+        if not os.path.exists(self.file_path):
+            self._save_data([])
+
+    def _load_data(self) -> List[dict]:
+        """JSON 파일에서 좋아요 목록 로드"""
+        try:
+            with open(self.file_path, "r", encoding="utf-8") as f:
```

Ruff (0.14.13)

Unnecessary mode argument

Remove mode argument

(UP015)